multi-node/generic/controller-install.sh

```diff
@@ -33,6 +33,9 @@ export DNS_SERVICE_IP=10.3.0.10
 # Whether to use Calico for Kubernetes network policy.
 export USE_CALICO=false
 
+# Determines the container runtime for kubernetes to use. Accepts 'docker' or 'rkt'.
+export RUNTIME=docker
```

Comment:
nit, `CONTAINER_RUNTIME` seems clearer.

For consistency, should this be `K8S_CONTAINER_RUNTIME` (to match the network-plugin one)?